element-hq/element-x-android
# Update timeline items read receipts when the room members are loaded

## Type of change


## Content

When we fetch an item from the timeline cache, we now always update the read receipts with the latest data from the room members list.

## Motivation and context

Fixes #2176 .

## Tests

- Open a room (ideally a small-ish one so it doesn't take too long to fetch the room members list).
- Check the read receipts.

If after a few seconds they change and now contain the right images and user names, it's working.

## Tested devices

- OS version(s): 14

## Checklist
